modules/nw-configuring-router-compression.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+[id="nw-configuring-router-compression_{context}"]
+= Using router compression
+
+You configure the HAProxy Ingress Controller to specify router compression. You can use the `CompressionMIMEType` variable to define the formats of MIME types to which compression is applied. The types are: application, image, message, multipart, text, video, or a custom type prefaced by "X-".
```

Comment:
`You can use the `CompressionMIMEType` variable` should be `You can use the `mimeTypes` variable`